src/test/java/edu/cornell/kfs/fp/batch/service/impl/AdvanceDepositServiceImplTest.java

```diff
@@ -641,6 +641,11 @@ public PersistableBusinessObject retrieve(PersistableBusinessObject object) {
         }
 
         @Override
+        public <T extends BusinessObject> Collection<T> findMatching(Class<T> aClass, Map<String, ?> map, int i, int i1, String[] strings) {
```

Comment:
I think it's fine. It's in a MockBusinessObjectService just used for a test class. This method was recently added, so I needed to implement it to fix the compiler error, but most of the methods in that Mock, included another findMatching version, return null.